sling/nlp/parser/trainer/pytorch_modules.py

```diff
@@ -553,6 +553,7 @@ def link(bldr, name, dim, cnx, prefix=True):
     # Add link and connector for previous FF steps.
     ff_cnx = ff.cnx("step", args=[])
     ff_steps = link(ff, "steps", spec.ff_hidden_dim, ff_cnx, False)
+    ff_cnx.add(fl.var("ff/hidden:0"))
```

Comment:
I see, so all this while ff/steps was set to the right data (written by ff/hidden), but their alignments were not guaranteed to be the same?